package.json

```diff
@@ -2,114 +2,60 @@
   "name": "tify",
   "version": "0.23.1",
   "description": "TIFY is a slim and mobile-friendly IIIF document viewer.",
-  "homepage": "https://github.com/tify-iiif-viewer/tify",
-  "repository": {
-    "type:": "git",
-    "url": "https://github.com/tify-iiif-viewer/tify"
-  },
-  "bugs": {
-    "url": "https://github.com/tify-iiif-viewer/tify/issues"
-  },
-  "author": {
-    "name": "Tobias Schäfer",
-    "url": "https://aspectis.net/"
-  },
-  "license": "AGPL-3.0",
+  "contributors": "https://github.com/tify-iiif-viewer/tify/graphs/contributors",
   "scripts": {
-    "prepare": "rimraf openseadragon && git clone https://github.com/openseadragon/openseadragon.git && cd openseadragon && git checkout tags/v2.4.1 && cd .. && npm run fix-openseadragon",
+    "serve": "vue-cli-service serve",
+    "serve-mock-api": "node tests/iiif-api/server.js",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "lint-fix": "vue-cli-service lint --fix",
+    "e2e": "node tests/e2e/runner.js chrome",
     "fix-openseadragon": "replace \"\\}\\(this, function \\(\\) \\{\" \"}(window, function () {\" ./openseadragon/src/openseadragon.js",
-    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs",
-    "build": "node build/build.js",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "unit-watch": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js",
-    "e2e": "ELECTRON_DISABLE_SECURITY_WARNINGS=true node test/e2e/runner.js",
+    "postversion": "git push && git push --tags && npm run update-gh-pages && echo Tag v$npm_package_version created. Run \"npm publish\" to publish \"$npm_package_name\" on npmjs.com.",
+    "prepare": "rimraf openseadragon && git clone https://github.com/openseadragon/openseadragon.git && cd openseadragon && git checkout tags/v2.4.1 && cd .. && npm run fix-openseadragon",
+    "preversion": "rimraf node_modules && npm install && npm run test",
     "test": "npm run unit && npm run e2e",
+    "unit": "vue-cli-service test:unit",
     "update-gh-pages": "git checkout gh-pages && git rebase master && git push -f origin gh-pages && git checkout master",
-    "preversion": "rimraf node_modules && npm install && npm run test",
-    "version": "npm run build && git add dist",
-    "postversion": "git push && git push --tags && npm run update-gh-pages && echo Tag v$npm_package_version created. Run \"npm publish\" to publish \"$npm_package_name\" on npmjs.com."
+    "version": "npm run build && git add dist"
   },
-  "devDependencies": {
-    "autoprefixer": "^9.7.1",
+  "dependencies": {
```

Comment:
All dependencies should go to devDependencies as we don't have any runtime deps

Reply:
done